src/picknik_ur_base_config/config/config.yaml

```diff
@@ -11,13 +11,12 @@ hardware:
   # You can change this to another UR model but you must update any configuration affected by the different arm size.
   # UR models in the ur_description package are ur3, ur3e, ur5, ur5e, ur10, ur10e, and ur16e.
   # [Required]
-  type: ${UR_HARDWARE_TYPE:-ur5e}
+  type: ur5e
```

Comment:
Do you think we can make this parameter not required? 
This parameter makes sense only for switching between different URs

Reply:
I agree we should drop this, not only is it UR specific but the UR macro uses the key `name` instead.

[In this example](https://github.com/PickNikRobotics/moveit_studio_ur_ws/pull/301/files) I've replaced the `picknik_ur` macro with the `ur_description`'s `ur_robot` macro so that I can use multiple different URs in my config. This is not possible using the `type:` tag in `config.yaml`

Reply:
If we're going to make schema changes to the config.yaml like this (which I think we should do!) I'm going to go make a different PR in Pro that uses Pydantic to read and validate our YAML files so we have a better idea of the actual shape of the data before we start making changes.